services/core-web/common/components/PartyAppointmentTable.js

```diff
@@ -104,10 +105,7 @@ const PartyAppointmentTable = (props) => {
       dataIndex: "dates",
       render: (text, record) => (
         <div title="Dates">
-          {record.startDate}
-          {' '}
--
-          {record.endDate}
+          {record.startDate} -{record.endDate}
```

Comment:
this is pretty minor but I think there's a space missing after the '-' before the end date

Reply:
Nice catch. And I just fixed this.